src/applications/personalization/profile-2/components/MHVTermsAndConditionsStatus.jsx

```diff
@@ -11,12 +10,12 @@ const MHVTermsAndConditionsStatus = ({ mhvAccount }) => {
 
   if (mhvAccount.termsAndConditionsAccepted) {
     return (
-      <>
+      <div className="vads-u-display--flex vads-u-flex-direction--column">
```

Comment:
Just curious if this is actually needed here?

Reply:
Leaving it out results in 
![image](https://user-images.githubusercontent.com/14869324/88067773-a4141300-cb2c-11ea-95f3-1dc458e08372.png)

Reply:
Oh duh it was a fragment before, not a `div`. I think just converting it to a `div` and leaving off the `flex` stuff would still make it look since the `div` would automatically stack its child block elements vertically but this is fine, too!